Apps/APN/package.json

```diff
@@ -8,21 +8,24 @@
     "bundle:ios": "react-native bundle --entry-file index.js --platform ios --dev false --bundle-output ios/main.jsbundle --assets-dest ios",
     "clean-install": "./scripts/clean-install-dependencies.sh",
     "clean-install-android": "./scripts/clean-run-android.sh",
-    "clean-install-ios": "./scripts/clean-run-ios.sh"
+    "clean-install-ios": "./scripts/clean-run-ios.sh",
+    "dev:update": "../../scripts/dev:update.sh",
+    "postinstall": "pod update --project-directory=ios",
+    "preinstall": "npm run dev:update"
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
     "@react-native-clipboard/clipboard": "^1.11.2",
     "@react-native-community/push-notification-ios": "^1.10.1",
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
-    "customerio-reactnative": "../..",
+    "customerio-reactnative": "file:../../customerio-reactnative.tgz",
     "expo": "~42.0.1",
     "expo-splash-screen": "~0.11.2",
     "expo-status-bar": "~1.0.4",
     "expo-updates": "~0.8.1",
     "react": "18.2.0",
-    "react-dom": "16.13.1",
+    "react-dom": "^18.2.0",
```

Comment:
Updated this because npm was giving me a lot of warnings about dependencies not matching up.